flink-clients/src/main/java/org/apache/flink/client/CliFrontend.java

```diff
@@ -637,20 +637,25 @@ protected int savepoint(String[] args) {
 								"Specify a Job ID to trigger a savepoint."));
 			}
 
-			return triggerSavepoint(options, jobId);
+			String savepointDirectory = null;
+			if (cleanedArgs.length == 2) {
```

Comment:
Changed the check to `>= 2` and printed a message that some arguments are unneded.